app/assets/stylesheets/color_themes/_color_theme_override_origwhite.scss

```diff
@@ -0,0 +1,26 @@
+body {
+  #main_stream .stream_element {
+    border: 0;
+    border-bottom: 1px solid $border-grey;
+    margin-bottom: 10px;
+  }
+
+  .profile_header > div {
+    border-bottom: 1px solid $border-grey;
+  }
+
+  .left-navbar .hoverable { border-bottom: 0; }
+
+  .profile-sidebar,
+  .left-navbar-fixed-background,
+  &.page-tags .left-navbar-fixed-background,
+  .left-navbar {
+    border-right: 1px solid $border-grey;
```

Comment:
I tried that but it looked weird without changing some padding/margin of the stream / the profile header. I'm not planning to change this kind of stuff since these changes are much more likely to break stuff later when we adjust the original "theme". I'd also like to emphasis that we are talking about a "color theme" here.